### DISCUSSION TITLE
Data Compression Chapter

### DISCUSSION THREAD

**Discussion by nush320** [biostars/biostar-handbook-issues#266] [Q&A]
Hi, I am confused about how to practice the commands in the Data Compression chapter. Am I supposed to first download the compressed files from somewhere? Or just make directories with the compressed files by myself?

Thank you in advance.

**Comment by ialbert** [comment]
This is a good point, maybe I will modify this chapter to make it simpler to get started.

If you want to practice the process scroll down on the page to the section: "How do I compress or uncompress multiple files"

**Reply by nush320** [reply]
Thanks!
